lib/internal/crypto/diffiehellman.js

```diff
@@ -219,21 +219,16 @@ function encode(buffer, encoding) {
 }
 
 function getFormat(format) {
-  let f;
   if (format) {
     if (format === 'compressed')
-      f = POINT_CONVERSION_COMPRESSED;
+      return POINT_CONVERSION_COMPRESSED;
     else if (format === 'hybrid')
```

Comment:
The else becomes obsolete here. The same for the next one.

Reply:
Good point, I missed that. Will update shortly. Thanks